Lib/opcode.py

```diff
@@ -67,7 +67,6 @@ def jabs_op(name, op):
 def_op('UNARY_NOT', 12)
```

Comment:
This diff looks worse than it really is.

Some opcodes were out-of-order or had incorrect vertical whitespace, which meant that my new opcodes actually shadowed / were shadowed by other opcodes when I first added them. It was confusing (to say the least), so I fixed it.